src/NzbDrone.Core.Test/Datastore/DatabaseRelationshipFixture.cs

```diff
@@ -59,21 +59,21 @@ public void embedded_document_as_json()
         {
             var quality = new QualityModel { Quality = Quality.Bluray720p, Revision = new Revision(version: 2 )};
 
-            var history = Builder<History.History>.CreateNew()
+            var history = Builder<History.EpisodeHistory>.CreateNew()
```

Comment:
Can actually get rid of `History` now, here and in other places. Might be worth to just do a search&replace for `History.EpisodeHistory`.

Reply:
Yeah, I though ti got them, but I think those were the `Core.History.EpisodeHistory` ones, replaced them now.